### PR TITLE
remove_deprecated_part_4

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi @tinatian , @shuyangzhou,

I have removed deprecated methods or classes for "portal-impl/src/com/liferay/portal/upgrade", "portal-impl/src/com/liferay/portal/upload", "portal-impl/src/com/liferay/portal/util", "portal-impl/src/com/liferay/portlet/admin/util", "portal-impl/src/com/liferay/portal/verify", "portal-impl/src/com/liferay/portlet/asset/service/permission"
Thanks for checking. see https://github.com/jeffwu0724/liferay-portal/pull/27.

Thanks,
Jeff

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:test

**Comment by jeffwu0724** [comment]
@kyle-miho Hey Kyle, could you please help me to scan the test result?

Thanks,
Jeff

File: portal-impl/src/com/liferay/portal/util/PropsValues.java [hunk 1]
```diff
@@ -181,14 +181,6 @@
 	public static final String AUTH_LOGIN_PORTLET_NAME = PropsUtil.get(
 		PropsKeys.AUTH_LOGIN_PORTLET_NAME);
 
-	/**
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/util/PropsValues.java:184]: @jeffwu0724 For all the properties you removed here, you should also remove them from portal.properties and add them to VerifyProperties. Please do:
1. Check all the properties removed here on master, if the usages are all from deprecated methods/classes, remove the deprecated methods/classes first, then remove the properties.
2. Send me a pull for 1 to remove properties.
3. Rebase the changes in this pull, send me a pull to remove deprecate as normal.

**Comment by tinatian** [comment]
@jeffwu0724 Please see my comments and resend.

Thanks.
Tina.

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**